### ISSUE TITLE
[queries] Resubmissions

### ISSUE THREAD

**Issue by krobasky** [RENCI/dashboard#16]
Number of Resubmissions

Breakout of "each of the above" (the items in #15) by funding status (Grant not submitted, Grant submitted [further breakout when grant is submitted: funded, not funded, funding pending])

Items in #15:
- [x] Overall
- [ ]       By Fiscal Year (July 1, 2016 – June 30, 2017 (Year1), July 1, 2017 – June 30, 2018 (Year2) . . . )
- [ ]       By Month
- [x]       By Institution submitted
- [x]       By Institution assigned to (i.e. which TIC/RIC)
- [x]       By Therapeutic Area

**Comment by krobasky** [comment]
@mbwatson, go ahead and close this ticket if the PMD already answers every question.

If the PMD only answers some of the questions posed in this ticket, check those boxes & please work with @rajshruti18 to identify what will be necessary to answer the rest.

If the question itself is unclear, even after speaking with Shruti and @xu-hao, please make a trello card on the 'Challenges' board to ensure we address the question at the next meeting.

**Comment by mbwatson** [comment]
I'm writing to make a note (for myself) that I've reached out to @rajshruti18 for help on this one.

**Comment by rajshruti18** [comment]
Below is the query to extract resubmitted proposals [resulting columns: proposal_id, services_approved, funding]:
SELECT proposal.proposal_id, name2.description as services_approved,service_services_approved, name3.description as funding, funding.funding 
FROM proposal 
INNER JOIN service ON proposal.proposal_id=service.proposal_id 
INNER JOIN funding ON funding.proposal_id=service.proposal_id 
INNER JOIN service_services_approved ON proposal.proposal_id=service_services_approved.proposal_id
INNER JOIN name as name3 on name3.index=cast(funding.funding as varchar) and name3.”column”=’funding’
INNER JOIN name as name2 on name2.id=service_services_approved.services_approved and name2.”column”=’services_approved’
WHERE 
proposal.redcap_repeat_instrument is null and service.redcap_repeat_instrument is null and proposal.redcap_repeat_instance is null and service.redcap_repeat_instance is null and funding.redcap_repeat_instrument is null and funding.redcap_repeat_instance is null AND proposal.redcap_repeat_instance is null AND proposal.redcap_repeat_instrument is null and proposal.protocol_status='21'
order by proposal.proposal_id;

**Comment by rajshruti18** [comment]
Count of resubmitted proposals by institution submitted:
SELECT COUNT(*),name2.description as org_name 
FROM proposal
INNER JOIN name as name2 on name2.index=cast(proposal.org_name as varchar) and name2."column"='org_name'
WHERE
proposal.redcap_repeat_instrument is null and proposal.redcap_repeat_instance is null and proposal.protocol_status='21'
GROUP BY name2.description;

**Comment by rajshruti18** [comment]
**Count of resubmitted proposals by institution assigned to:**
SELECT COUNT(*),name2.description as tic_ric_assign
FROM proposal
INNER JOIN name as name2 on name2.index=cast(proposal.tic_ric_assign_v2 as varchar) and name2."column"='tic_ric_assign_v2'
WHERE
proposal.redcap_repeat_instrument is null and proposal.redcap_repeat_instance is null and proposal.protocol_status='21'
GROUP BY name2.description;

**Comment by rajshruti18** [comment]
**Count of resubmitted proposals by therapeutic area:**
SELECT COUNT(*),name2.description as therapeutic_area
FROM proposal
INNER JOIN study on proposal.proposal_id=study.proposal_id
INNER JOIN name as name2 on name2.index=cast(study.theraputic_area as varchar) and name2."column"='theraputic_area'
WHERE
proposal.redcap_repeat_instrument is null and proposal.redcap_repeat_instance is null and proposal.protocol_status='21'
GROUP BY name2.description;

**Comment by rajshruti18** [comment]
**Number of resubmitted proposals (overall):**
SELECT COUNT(*)
FROM proposal
WHERE
proposal.protocol_status='21';

**Comment by mbwatson** [comment]
Great, thanks!

**Comment by krobasky** [comment]
@mbwatson what about 'fiscal year' and 'month'?

**Comment by mbwatson** [comment]
@krobasky You're right--I closed the wrong one! Opening this up again now. Sorry about the confusion!
